### DISCUSSION TITLE
Modifying Chat Window Colors

### DISCUSSION THREAD

**Discussion by mitchkesller** [nicobrauchtgit/obsidian-Smart2Brain#128] [Q&A]
How can I change the colors of the AI chat window to make it more legible? Pretty green at any kind of coding, are there any templated css files we could use to alter? Or is there a setting I'm missing somewhere?

<img width="405" alt="Screenshot 2024-05-16 at 10 29 45 PM" src="https://github.com/your-papa/obsidian-Smart2Brain/assets/88554825/493a5df3-85a3-4ff7-bc25-153b8371d64f">

**Comment by nicobrauchtgit** [comment]
Hi @mitchkesller, first of, nice theme! We currently don't have the option to change the colors manually, however the chat definitely should not look like this and is probably a bug in our code. Could you share which theme you are using so we can investigate? 
Thanks😊
